app/src/main/res/values-de/strings.xml

```diff
@@ -40,10 +41,12 @@
     <string name="use_safe_stores">Sichere Stores nutzen (Aptoide)</string>
     <string name="root_install">Root-Installation</string>
     <string name="source_apkmirror" translatable="false">ApkMirror</string>
-    <string name="source_fdroid" translatable="false">F-Droid (Main)</string>
-    <string name="source_izzy" translatable="false">F-Droid (Izzy)</string>
+    <string name="source_fdroid" translatable="false">F-Droid Offiziell</string>
+    <string name="source_izzy" translatable="false">F-Droid Izzy</string>
```

Comment:
```suggestion
    <string name="source_fdroid" translatable="false">F-Droid (Main)</string>
    <string name="source_izzy" translatable="false">F-Droid (Izzy)</string>
```
Again, you shouldn't modify those.

Reply:
Just look at the main strings.xml in the values folder. 
Those strings were also changed and have the brackets removed.

Reply:
Yes, that's fair, but you shouldn't put "Officiziell". Keep it as-is with out the brackets.

Reply:
Maybe I should mark these as translatable.